service/history/task/task.go

```diff
@@ -56,11 +57,22 @@ func isRedispatchErr(err error) bool {
 	return errors.As(err, &redispatchErr)
 }
 
+const (
+	activeTaskRedispatchInitialInterval  = 5 * time.Second
+	standbyTaskRedispatchInitialInterval = 30 * time.Second
```

Comment:
what exactly are you planning to compare in metrics by increasing standby redispatch interval to 30s?

Reply:
The number of standby tasks being executed per second. Currently, the qps of some standby tasks increases and some decreases.